docs/download.md

```diff
@@ -73,6 +73,10 @@ New functionality for this release includes:
   * Range windows supporting non-timestamp order by expressions
 * Enabling large joins that can spill out of memory
 * Support for the `concat_ws` operator
+* Spark Qualification and Profiling Tools
+  * A qualification tool to look at a set of applications to determine if the RAPIDS Accelerator for
```

Comment:
I think there should be some language that it's based on Spark event logs to set the right expectations